output/csvwriter.go

```diff
@@ -1 +1,68 @@
 package output
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"reflect"
+	"strings"
+)
+
+type CsvWriter struct {
+	file *os.File
+}
+
+func (c *CsvWriter) Init(filename string) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	c.file = file
+	err = writeHeaders(c.file)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func writeHeaders(file io.Writer) error {
```

Comment:
should be public, it is part of interface Writer
and should have string parameter

parameter should be pointer

Reply:
see comment below - also, what do you think about my wuestions above?